test-harness/specs/security-scheme-validation-AC-10.oas2.txt

```diff
@@ -0,0 +1,34 @@
+====test====
+Adding a required header of "Authorization: Bearer <<BEARER_TOKEN>>" results in granted access.
```

Comment:
Would be nice to have a test checking 'denied' access for oauth2. What do you think?

Reply:
it sure would :) will add :+1:

Reply:
added in https://github.com/stoplightio/prism/pull/484/commits/4a0fd4990a575d7a42d26ec28cc5e62dba5c0719#diff-ddb070efafba3257e4c1ff0884ad6463